config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -410,7 +411,7 @@ EOF
         # docker-in-docker needs privileged mode
         securityContext:
           privileged: true
-$(resources_for_kubernetes "$kubernetes")
+$(resources_for_kubernetes "$kubernetes" "    ")
```

Comment:
To expand on the previous point: why is this the right amount of spaces? This becomes more obvious when it matches the number of spaces before `securityContext`.

Reply:
👍🏼 done